### DISCUSSION TITLE
UTXO  Problem

### DISCUSSION THREAD

**Discussion by Bo390** [DeFiCh/app#978] [Q&A] [answered]
I have 2.59 UTXO in my balance. I am trying to send some DFI to Cake but I get a "Insufficient UTXO DFI for transaction fees" error. I have tried restarting the app (i'm on 2.5.1) and I also tried sending myself some UTXO from the UTXO Recharger on https://mydeficha.in/. This has not helped and I am still unable to send to Cake. Can anyone advise? Thanks.

**Comment by izzycsy** [comment, marked as answer]
https://github.com/DeFiCh/app/discussions/846

**Reply by Bo390** [reply]
Using the example given I tried to send my gettokenbalances total amount of DFI to one of my two addresses. It says not enough UTXO even though using listaddressgroupings I see 2.58 UTXO on that exact same address.

**Comment by Bo390** [comment]
After updating to 2.5.2. I was able to send my DFI so no idea what the problem was.
